schema/geometry_schema.json

```diff
@@ -1,5 +1,6 @@
 {
 	"$schema": "http://json-schema.org/draft-04/schema#",
+	"version": "0.3.0",
```

Comment:
@macumber Ah, that's a useful addition as well. But I was thinking we would want a version *on the schema*. It looks like your PR adds lines that say "a valid floorplan adhering to this schema will have a version". I'd like to also say "you're looking at version 0.3.0 of the schema".

Happy to roll this part back though if you think it doesn't make sense.